src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -36,176 +36,21 @@
         </ResourceDictionary>
     </Page.Resources>
 
-    <ScrollViewer ViewChanging="ViewChanging">

```

Comment:
Putting a ListView inside a ScrollViewer messes things up, so the outer ScrollViewer has been removed in favour of a Grid. Ultimately the only region we want scrolling for anyway is the list of color schemes so this works out